client/src/components/SponsorsComps/HomePageSponsorsSection.jsx

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable prettier/prettier */
+import React from "react";
+import HomeCommunitySponsors from "./HomeCommunitySponsors";
+import styled from 'styled-components';
+import {
+  Container,
+  BreakLine,
+} from '../GlobalStyledComponents/ReversimStyledComps'
+import { SponsorMiniPremium } from './SponserGeneralComps';
+
+// Styled-components section
+
+const SponserSectionContainter = styled(Container)`
+  margin-top: 80px;
+  margin-bottom: 100px;
+  flex-direction: column;
+  flex-wrap: nowrap;
+`;
+
+const HeadingAlinger = styled.div`
+  ${ ( { theme: { space } } ) => `
+    width: 100%;
+    display: flex;
+    align-items: baseline;
+    margin-bottom: ${space.xxl};
+  `}
+`;
+
+const Heading = styled.h2`
+  ${ ( { theme: { color, font } } ) => `
+        width: inherit;
+        position: relative;
+        z-index: 1;
```

Comment:
I dont think we need 🤔

```
position: relative;
z-index: 1;
```

I removed it in developer and it looks fine 

Im Im not mistaken of course ... lets always aim for the minimum css needed